eng/pipelines/templates/stages/platform-matrix-cryptography-dependency.json

```diff
@@ -7,7 +7,7 @@
       "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" },
       "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
     },
-    "PythonVersion": [ "pypy3.7", "3.8"],
```

Comment:
python 3.9 is present `included` in the matrix from line 36.

Reply:
Oh, thank you for the catch! I removed 3.9 from this file. Thank you.